src/main/java/lotto/constant/LottoValue.java

```diff
@@ -0,0 +1,19 @@
+package lotto.constant;
+
+public enum LottoValue {
+    LOTTO_PRICE(1000),
```

Comment:
가격은 천단위 구분자를 이용하면 더 가독성이 좋아질 것 같습니다!
```suggestion
    LOTTO_PRICE(1_000),
```